src/pages/DelegatoryValidator/MyDepositsSection.tsx

```diff
@@ -137,11 +154,29 @@ export default function MyDepositsSection({
           <GeneralTableRow>
             {columns.map((deposit) => {
               const Cell = MyDepositsCells[deposit];
-              return <Cell key={deposit} accountResource={accountResource} />;
+              return (
+                <Cell
+                  key={deposit}
+                  accountResource={accountResource}
+                  handleClickOpen={handleClickOpen}
+                />
+              );
             })}
           </GeneralTableRow>
         </GeneralTableBody>
       </Table>
+      {connected ? (
```

Comment:
I remember I saw this logic before, is this a reusable component?

Reply:
its reused [here](https://github.com/aptos-labs/explorer/blob/25307f23fe1ef6d16f8182d588afb098e9a1e769/src/pages/DelegatoryValidator/StakingBar.tsx#L110) but its for staking flow and this is for unstaking flow.